CodeEdit/Features/InspectorSidebar/Models/FileInspectorModel.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  FileInspectorModel.swift
+//  CodeEdit
+//
+//  Created by Nanashi Li on 2022/04/17.
+//
+
+import SwiftUI
+
+public final class FileInspectorModel: ObservableObject {
```

Comment:
I think this file has been deleted in the recent inspector PR

Reply:
You're right.

Reply:
It's deleted now.